editor/state.js

```diff
@@ -303,6 +303,21 @@ export function isSidebarOpened( state = false, action ) {
 	return state;
 }
 
+export function editorWidth( state = null, action ) {
+	const layout = document.querySelector( '.editor-layout__content' );
```

Comment:
Also, every reducer is called for every action, so these query selectors are executed very often, and almost always needlessly so. At the very least we should check `action.type` before performing any queries.